docs/server/source/production-deployment-template/server-tls-certificate.rst

```diff
@@ -29,8 +29,27 @@ You can create the server private key and certificate signing request (CSR)
 by going into the directory ``member-cert/easy-rsa-3.0.1/easyrsa3``
 and using something like:
 
+.. note::
+
+    Make sure you are fullfilling the following requirements for MongoDB server/member certificates:
+
+    * A single Certificate Authority (CA) must issue all the x.509 certificates for the members of a sharded cluster
+      or a replica set.
+    * The Distinguished Name (DN), found in the member certificate’s subject, must specify a non-empty
+      value for at least one of the following attributes: Organization (O), the Organizational Unit (OU)
+      or the Domain Component (DC).
+    * The Organization attributes (O‘s), the Organizational Unit attributes (OU‘s), and the Domain Components (DC‘s)
+      must match those from the certificates for the other cluster members. To match, the certificate must match
+      all specifications of these attributes, or even the non-specification of these attributes. The order of the
+      attributes does not matter.
+    * Either the Common Name (CN) or one of the Subject Alternative Name (SAN) entries must match the hostname of the
+      server, used by the other members of the cluster.
+
+   For more information about the requirements for MongoDB server/member certificates, please consult the
+   `official MongoDB documentation <https://docs.mongodb.com/manual/tutorial/configure-x509-member-authentication>`_.
+
```

Comment:
Can we point to the MongoDB docs, rather than copy it here?
As MongoDB _might_ change its instructions, and we do not want to end up with wrong instructions here.